src/lib/frontend/stores/shoppingCartStore/ShoppingCartStore.ts

```diff
@@ -85,18 +85,6 @@ export function decreaseProductSelectedQuantityFromShoppingCartStore(
     }
 }
 
-export function removeProductFromShoppingCartStore(id: string): void {
```

Comment:
Since we don't have a remove button anymore, the only way to remove a product from the cart is by decreasing the selected quantity to 0. I noticed you about it in this message 
<img width="576" alt="image" src="https://github.com/6C-Solutions-4-Small-Businesses/online-shop-template/assets/46734336/ae74a304-dcbd-436a-b390-03f15b71f6f2">

Reply:
So the item will disappear from the cart if I set the quantity to 0?

Reply:
yes

Reply:
I have created this issue: https://github.com/6C-Solutions-4-Small-Businesses/online-shop-template/issues/55

Can you add an animation to prevent a brutal disparition of the item?